### PR TITLE
Document topk/bottomk becoming aggregators.

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/docs/querying/operators.md [hunk 1]
```diff
@@ -186,18 +186,24 @@ vector of fewer elements with aggregated values:
...
+clause, even if their label values are identical between all elements of the
+vector. The `keep_common` clause allows keeping those extra labels (labels that
+are identical between elements, but not in the `by` clause).
+
+`topk` and `bottomk` are different from other aggregators in that a subset of
+the input samples, including the original labels, are returned in the result
+vector. `by` and `without` are only used to bucket the input vector.
 
 Until Prometheus 0.14.0, the `keep_common` keyword was called `keeping_extra`.
 The latter is still supported, but is deprecated and will be removed at some
```
Comment (fabxc) [inline content/docs/querying/operators.md]: Seeing this sentence: can we remove `keeping_extra` for 1.0?

Comment (brian-brazil) [inline content/docs/querying/operators.md]: You mean the PR I sent after this one that you just merged?

Comment (fabxc) [inline content/docs/querying/operators.md]: Ha, my bad – guess I was focused on the old alerting keywords.
Then we just need to adjust the docs.

Comment (brian-brazil) [inline content/docs/querying/operators.md]: I was going to do that in a different PR

**Comment by fabxc** [comment]
👍
